packages/core/content-releases/admin/src/components/tests/CMReleasesContainer.test.tsx

```diff
@@ -59,8 +59,10 @@ describe('CMReleasesContainer', () => {
   it('should render the container', async () => {
     render();
 
-    const informationBox = await screen.findByRole('complementary', { name: 'Releases' });
-    expect(informationBox).toBeInTheDocument();
+    await waitFor(async () => {
```

Comment:
I think findBy queries already use a waitFor https://testing-library.com/docs/dom-testing-library/api-async#findby-queries. I think what you end up doing here with an additional waitFor is increase the timeout / retries. But It looks like you can pass waitFor options to the findBy query to do that as well.

Reply:
Mark is right, it'll also fail if it's not in the document. There's probably a different `async` fetch happening elsewhere that updates the component, you might needed to await something else.